apps/vaporgui/VLineItem.cpp

```diff
@@ -2,12 +2,30 @@
 #include <QHBoxLayout>
 #include <QLabel>
 
-VLineItem::VLineItem(std::string label, QWidget *rightWidget)
+#define LEFT_MARGIN    0
+#define TOP_MARGIN     0
+#define RIGHT_MARGIN   0
```

Comment:
These are defined in multiple locations

Reply:
Fixed.

Reply:
It still appears to be defined in both VContainer and VLineItem